### PR TITLE
New diagram added to main page

### PR DESCRIPTION
**Pull request by denkojonasz**

### CODE REVIEW

**Review by tymik — APPROVED** [review]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ inspiration on this site and your life is going to be at least 1% easier!
 ## Why is this 1% so important?
 
 There's an excellent article about it, [you can read it here][1],
-but let's summarize it with only one picture:
+but let's summarize it with only two pictures:
 
-![Marginal gains][image-1]
+<div align="center"><img src="assets/marginal-gains.jpg" /></div>
+<br/>
+<div align="center"><img src="assets/image.png" /></div>
```
Comment (lubieniebieski) [inline docs/index.md:29]: seems like the `image.png` is missing

**Review by lubieniebieski — CHANGES_REQUESTED** [review]
please provide a valid image file with this PR

**Comment by lubieniebieski** [comment]
sorry, my bad regarding the missing file, but please provide a better, self-explaining name for the file instead

**Pull request merged**